projects/ngx-drag-to-select/package.json

```diff
@@ -1,11 +1,11 @@
 {
   "name": "ngx-drag-to-select",
-  "version": "2.1.0",
+  "version": "2.2.0",
```

Comment:
Whoops, this shouldn't be updated. I'll bump the version number once I cut the new release and publish it on npm.